components/button/button-subtle.js

```diff
@@ -208,7 +208,7 @@ class ButtonSubtle extends ButtonMixin(LitElement) {
 				</span>
 			</button>
 			${this.description ? html`<span id="${this._describedById}" hidden>${this.description}</span>` : null}
-			${this.disabled && this.disabledTooltip ? html`<d2l-tooltip for="${this._buttonId}">${this.disabledTooltip}</d2l-tooltip>` : ''}
+			${this.disabled && this.disabledTooltip ? html`<d2l-tooltip class="vdiff-target" for="${this._buttonId}">${this.disabledTooltip}</d2l-tooltip>` : ''}
```

Comment:
Aside: I love how easy it's been to "fix" this kind of thing -- awesome job with that feature!

Reply:
Thanks, I've also been really happy with it!